### PR TITLE
Add additional output to log files

### PR DESCRIPTION
**Pull request by mfeurer**
This PR adds logging for
* the system information
* installed packages
* arguments passed to Auto-sklearn
* available components

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -4,5 +4,6 @@ recursive-include autosklearn/metalearning/files *.txt
 include autosklearn/util/logging.yaml
 recursive-include autosklearn *.pyx
```
Comment (franchuterivera) [inline MANIFEST.in:5]: This is legacy right? maybe we should add another issue to clean this one up

Comment (mfeurer) [reply MANIFEST.in:5]: Yes, indeed. Good point.

Comment (mfeurer) [reply MANIFEST.in:5]: https://github.com/automl/auto-sklearn/pull/934

**Review by franchuterivera — APPROVED** [review]
Ran the code and the log file works as expected!

**Pull request merged**